site_documentation.install

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Handle managing configuration and updating entity definition in an update hook instead of requiring manual deployment steps.